### PR TITLE
Rename and update master branch references to main

### PR DESCRIPTION
**Pull request by akchinSTC**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -66,4 +66,7 @@ yarn-error.log
 .pnp/
 .pnp.js
 # Yarn Integrity file
-.yarn-integrity
\ No newline at end of file
+.yarn-integrity
+
+#pycharm
```
Comment (ptitzler) [inline .gitignore:71]: Intentional changes?

File: src/pages/common-canvas/callbacks.mdx [hunk 2]
```diff
@@ -339,7 +339,7 @@ This callback is called before tips are shown for: palette items; nodes in the c
 * For ports, the port label is shown.
 * For links, no tip is shown by default.
 
-To overwrite the content, you can return either a string or a React element. If your code returns null for a particular type of tip, common-canvas will display the default tip for that object. For an example of an implementation for a tipHandler, see App.js in the test harness code (https://github.com/elyra-ai/canvas/blob/master/canvas_modules/harness/src/client/App.js).
+To overwrite the content, you can return either a string or a React element. If your code returns null for a particular type of tip, common-canvas will display the default tip for that object. For an example of an implementation for a tipHandler, see App.js in the test harness code (https://github.com/elyra-ai/canvas/blob/main/canvas_modules/harness/src/client/App.js).
```
Comment (ptitzler) [inline src/pages/common-canvas/callbacks.mdx:342]: Canvas repo is not renaming the `master` branch as part of this effort.

```suggestion
To overwrite the content, you can return either a string or a React element. If your code returns null for a particular type of tip, common-canvas will display the default tip for that object. For an example of an implementation for a tipHandler, see App.js in the test harness code (https://github.com/elyra-ai/canvas/blob/master/canvas_modules/harness/src/client/App.js).
```

File: src/pages/common-canvas/canvas-controller/canvas-controller-api.mdx [hunk 3]
```diff
@@ -152,7 +152,7 @@ setCategoryEmptyText(categoryId, emptyText)
 // nodeTypeObj - must conform to the style of node used by the palette as
 // described in the palette schema. See objects in nodeTypes array in the
 // palette schema:
-//  https://github.com/elyra-ai/pipeline-schemas/blob/master/common-canvas/palette/palette-v3-schema.json
+//  https://github.com/elyra-ai/pipeline-schemas/blob/main/common-canvas/palette/palette-v3-schema.json
```
Comment (ptitzler) [inline src/pages/common-canvas/canvas-controller/canvas-controller-api.mdx:155]:
```suggestion
//  https://github.com/elyra-ai/pipeline-schemas/blob/master/common-canvas/palette/palette-v3-schema.json
```

File: src/pages/common-canvas/canvas-controller/canvas-controller-api.mdx [hunk 4]
```diff
@@ -167,7 +167,7 @@ addNodeTypeToPalette(nodeTypeObj, categoryId, categoryLabel, categoryDescription
 // nodeTypeObjs - an array of nodetypes that must conform to the style of
 // nodes used by the palette as described in the palette schema. See objects
 // in nodeTypes array in the palette schema:
-//  https://github.com/elyra-ai/pipeline-schemas/blob/master/common-canvas/palette/palette-v3-schema.json
+//  https://github.com/elyra-ai/pipeline-schemas/blob/main/common-canvas/palette/palette-v3-schema.json
```
Comment (ptitzler) [inline src/pages/common-canvas/canvas-controller/canvas-controller-api.mdx:170]:
```suggestion
//  https://github.com/elyra-ai/pipeline-schemas/blob/master/common-canvas/palette/palette-v3-schema.json
```

**Review by ptitzler — CHANGES_REQUESTED** [review]
Need to coordinate with the canvas team.

File: .gitignore (hunk 1 above)
Comment (akchinSTC) [reply .gitignore:71]: yes

**Pull request merged**